data/src/main/java/io/gnosis/data/repositories/TokenRepository.kt

```diff
@@ -61,13 +62,13 @@ class TokenRepository(
 
     companion object {
         val ZERO_ADDRESS = Solidity.Address(BigInteger.ZERO)
-        val ETH_TOKEN_INFO = TokenInfo(
-            TokenType.ETHER,
+        val NATIVE_CURRENCY_INFO = TokenInfo(
```

Comment:
I think we should remove this NATIVE_CURRENCY_INFO in the near future as not every native currency has 18 decimals and this info should be delivered by client gw

Reply:
This seems to be related: https://github.com/gnosis/safe-client-gateway/issues/179